Source/PLCrashReporter.h

```diff
@@ -122,6 +122,7 @@ typedef struct PLCrashReporterCallbacks {
 + (PLCrashReporter *) sharedReporter PLCR_DEPRECATED;
 
 - (instancetype) initWithConfiguration: (PLCrashReporterConfig *) config;
+- (instancetype) initWithConfiguration: (PLCrashReporterConfig *) config basePath: (NSString *) basePath;
```

Comment:
Is there any reason we don't add this `basePath` to the configuration, ie as an additional property in `PLCrashReporterConfig`?
Having this extra parameter, while we already passing a config object, looks inefficient to me.
Also in all places where we use `basePath` we have the `configuration` parameter as well.

Reply:
Might be a good call actually, the only concern is that configuration only focuses on behavior-related preferences rather than paths. cc @MatkovIvan what's your take on this?